libraries/AP_HAL_ChibiOS/hwdef/scripts/STM32F412Cx.py

```diff
@@ -0,0 +1,347 @@
+#!/usr/bin/env python
```

Comment:
you should be able to use the existing STM32F412Rx.py instead of adding a new one. We could rename it to STM32F412xx.py. The extra pins that the Cx don't have do no harm.